package.json

```diff
@@ -24,6 +24,7 @@
     "webpack": "webpack --config webpack.config.js"
   },
   "dependencies": {
+    "bcash": "^1.1.0",
```

Comment:
Could probably be a peerDependency with the other two

Reply:
updated 👍